corehq/doctypemigrations/migrator_instances.py

```diff
@@ -36,6 +36,23 @@
     doc_types=('Domain',)
 )
 
+apps_migration = Migrator(
+    slug='apps',
+    source_db_name=None,
+    target_db_name=settings.NEW_DOMAINS_DB,
+    doc_types=(
+        'Application',
+        'ApplicationBase',
+        'CareplanConfig',
+        'DeleteApplicationRecord',
+        'DeleteFormRecord',
+        'DeleteModuleRecord',
+        'RemoteApp',
+        'SavedAppBuild',
+        'VersionedDoc',
```

Comment:
I'd say it's not strictly necessary, since there shouldn't be any docs with that doc type. If by some long-ago bug there were though, we'd want them to be moved to the new db, so I'd say it's fine to leave in there.